themes/custom/az_barrio/templates/layout/page.html.twig

```diff
@@ -130,9 +130,15 @@
 
           <div class="container">
             <div class="row">
-              {% if az_barrio_navbar_offcanvas %}
```

Comment:
I think we still need to this `if` statement to be here.  If we don't, the `endif` on line 157 should also be removed.